### ISSUE TITLE
Does image_generator shuffle data between epochs?

### ISSUE THREAD

**Issue by apt2000** [divamgupta/image-segmentation-keras#125]
Does the image_generator shuffle the data between epochs? It appears to me that it only shuffles it the first time it is created with a call to random.shuffle(). After this, the data simply cycles through via the call to itertools.cycle(). Is this  correct and is this the intended behaviour?

**Comment by divamgupta** [comment]
Yes, The data is shuffled only once.  I would add an option for shuffling after every epoch as well
